doc/programming_guide/asrc/resource_usage.rst

```diff
@@ -0,0 +1,134 @@
+
+**************
+Resource Usage
+**************
+
+Memory
+======
+
+Tile 0 memory usage in bytes:
+
+.. code-block:: console
+
+    Memory available:       524288,   used:      260976+.  MAYBE
```

Comment:
I'm guessing the tinyusb src code using function pointers is the reason for this. I don't see anything in the tools documentation that directly explains this. I'll change this section to give the approximate memory usage instead of exact numbers.